.dockerignore

```diff
@@ -1,6 +1,5 @@
 .git
 .build
 DerivedData
-Package.resolved
```

Comment:
This file (`Package.resolved`) should not be commited

Reply:
This should be added to Docker to ensure that the same version is deployed as tested with